app/grants/templates/grants/_new.html

```diff
@@ -155,7 +155,7 @@ <h1 class="text-center font-title-xl">Create a Grant</h1>
                   <label class="font-caption letter-spacing text-black-60 text-uppercase" for="twitter_handle_1">Project Twitter Handle</label>
                   <span class="font-smaller-6 badge badge-greylight text-capitalize ml-2">Required</span>
 
-                  <input id="twitter_handle_1" v-model="form.twitter_handle_1" name="twitter_handle_1" class="form__input form__input-lg" maxlength="20" type="text" placeholder="humanfund" required/>
+                  <input id="twitter_handle_1" v-model="form.twitter_handle_1" @blur.prevent="handleTwitterUsername($event)" name="twitter_handle_1" class="form__input form__input-lg" type="text" placeholder="humanfund" required/>
```

Comment:
I need to double check but I think for the `@blur.prevent=` part you can pass it the name of the function and not invoke it. `@blur.prevent="handleTwitterUsername` This applies to the other field as well. Double check me on this one though, I could be wrong here.

Reply:
I'll double check. I tried it both ways and it worked, but I noticed when I invoked it I saw less warnings in the console.

Reply:
What were the warnings?

Reply:
Actually, it doesn't apply now. I was seeing errors and warnings in the console earlier when I was using `window.getSelection()` to track the `input` field the user clicked on. That wasn't working properly in Chrome, so I used `event.target` instead, and now I see no errors or warnings.